lib/nolibc/Makefile.uk

```diff
@@ -52,6 +52,3 @@ LIBNOLIBC_SRCS-y += $(LIBNOLIBC_BASE)/random.c
 LIBNOLIBC_SRCS-$(CONFIG_LIBNOLIBC_SYSLOG) += $(LIBNOLIBC_BASE)/syslog.c
 
 LIBNOLIBC_SRCS-y += $(LIBNOLIBC_BASE)/qsort.c
-
-# Localize internal symbols (starting with __*)
-LIBNOLIBC_OBJCFLAGS-y += -w -L __*
```

Comment:
I do wonder why this was here in the first place. Having an `exportsyms.uk` would already localize all unlisted symbols. @skuenzer

Reply:
True, this is likely a pre-exportsyms leftover.